plugins/gcp/fix_plugin_gcp/resources/base.py

```diff
@@ -173,25 +173,33 @@ def _standard_edges(self, node: GcpResourceType, source: Optional[Json] = None)
             return True
 
         if source is not None:
+            if "zone" in source:
```

Comment:
```suggestion
            if ZoneProp in source:
```
to align with the other props